hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java

```diff
@@ -313,6 +313,7 @@ private Stream<SingleTransactionRecord> execute(@NonNull final UserTxn userTxn)
             }
             return finalRecordStream(userTxn);
         } catch (final Exception e) {
+            e.printStackTrace();
```

Comment:
This should not be necessary. The logger should print the stack trace.

Reply:
Ah yes it is a debug log. Thanks @netopyr . Will remove